config/routes.rb

```diff
@@ -55,6 +55,18 @@
         end
       end
 
+      if SETTINGS[:organizations_enabled]
+        resources :organizations, :only => [:index] do
+          resources :job_templates
```

Comment:
This should be probably limited to only `:index` and maybe `:show`. Without limiting it there are routes being generated for all 7 controller actions